CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,57 @@
+# Code of Conduct - Freedom of the Press Foundation
+
+Like the the broader technical and activism communities as a whole, the Freedom of the Press Foundation (FPF) team is made up of a mixture of staff and volunteers from all over the world, working on every aspect of our mission—including operations, software development, mentorship, and building connections with great people and organizations.
+
+To work together effectively in a large, diverse and open community, we have a few ground rules that we expect everyone to adhere to, be it FPF staff and board members, volunteers and event attendees; mentors, veterans, novices, or those seeking help and guidance.
+
+This isn’t an exhaustive list of things that you cannot do. Rather, take it in the spirit in which it’s intended—a guide to make it easier to enrich all of us and the communities in which we participate.
+
+This code of conduct applies to all spaces and events managed by the Freedom of the Press Foundation. This includes physical offices, GitHub repositories, online chat systems, the Support Forum, FPF hosted or sponsored events, and any other forums created by the Foundation which our team uses for communication. In addition, we take violations of this code outside these spaces into account when determining a person's ability to participate within them.
+
+If you believe someone is violating the code of conduct, we ask that you report it by talking to Emmanuel, or emailing **emmanuel@freedom.press**. If, for whatever reason, you are uncomfortable reporting the issue to Emmanuel, you may contact FPF’s Executive Director (**trevor@freedom.press**).  
+
+We are committed to taking fair, timely and appropriate corrective action against any violations of this code.
```

Comment:
No only this, we should say that we will not take action without the reporters consent, including releasing this information to others in the org. Someone might want to report something and have no action taken because of the problem that reporters of abuse can have their careers tanked as a result. I think that is very important to say because it's better to have the reports and not act on them than to make someone afraid to make a report.